levar-cli/src/main/scala/levar/LevarCli.scala

```diff
@@ -190,6 +195,35 @@ object LevarCli {
 
       args.subcommands match {
 
+        case List(args.helpCmd) => {
```

Comment:
This is really helpful, I wonder if we can engineer this more directly with [scallop](https://github.com/scallop/scallop)